src/components/ui/Rotation.module.css

```diff
@@ -1,7 +1,24 @@
-.action {
-	height: 40px;
+.container {
+	--ogcd-size: 30px;
```

Comment:
Doesn't look like you're using the cascade to override the var, probably better to use `@value` instead. It'll get compiled out :blobmorning: